cmd/neofs-node/config/morph/config.go

```diff
@@ -16,47 +15,36 @@ const (
 	// DialTimeoutDefault is a default dial timeout of morph chain client connection.
 	DialTimeoutDefault = 5 * time.Second
 
-	// PriorityDefault is a default endpoint priority for the morph client.
-	PriorityDefault = 1
-
 	// CacheTTLDefault is a default value for cached values TTL.
 	// It is 0, because actual default depends on block time.
 	CacheTTLDefault = time.Duration(0)
-
-	// SwitchIntervalDefault is a default Neo RPCs switch interval.
-	SwitchIntervalDefault = 2 * time.Minute
 )
 
 // RPCEndpoint returns list of the values of "rpc_endpoint" config parameter
 // from "morph" section.
 //
 // Throws panic if list is empty.
-func RPCEndpoint(c *config.Config) []client.Endpoint {
-	var es []client.Endpoint
+func RPCEndpoint(c *config.Config) []string {
```

Comment:
currently correct, but imo `[]client.Endpoint` is clearer and easier-to-support than `[]string` in code

Reply:
but `client.Endpoint` has been removed. do you mean keeping that string in `client` package with the only `string` field?

in fact, i do not like parsing internal structs from a config. that package should just read built-in types and return them IMO

Reply:
yeah, it's probably more accurate to say "maybe let's not purge `client.EndpointInfo` if it already exists?" I leave u to decide

about config package: i agree that it would make more sense to determine the type in place, because in fact they may well be different. Will be taken into account in future